internal/config/config_test.go

```diff
@@ -1693,6 +1693,120 @@ func TestParse(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "use ip pool selectors",
+			crs: ClusterResources{
```

Comment:
Can you add a test where we select a non existing label and we verify no advertisement is applied?

Reply:
done